hss/src/s6as6d_impl.cpp

```diff
@@ -2090,6 +2115,7 @@ void AIRProcessor::phase1()
       m_ans.add( m_dict.avpResultCode(), ER_DIAMETER_INVALID_AVP_VALUE);
       m_ans.send();
       StatsHss::singleton().registerStatResult(stat_hss_air, 0, ER_DIAMETER_INVALID_AVP_VALUE);
+      hssStats::Instance()->increment(hssStatsCounter::MME_MSG_TX_S6A_AUTH_INFO_ANSWER_FAILURE, {{"result_code","invalid_imsi"}});
```

Comment:
we can also add a counter for resync AIR.

Reply:
Great..Let me see.